### DISCUSSION TITLE
getting error unhandled runtime  "wcm-button"

### DISCUSSION THREAD

**Discussion by SumiBisht** [WalletConnect/walletconnect-monorepo#2966] [Developer Support]
Hey I am using walletconnect eth-provider but sometimes out of nowhere i am getting this error.
Unhandled Runtime Error
NotSupportedError: Failed to execute 'define' on 'CustomElementRegistry': the name "wcm-button" has already been used with this registry
<img width="770" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/40060905/28dcd0fb-7f28-4ab0-bd96-6b8492c7f91d">

**Comment by boidushya** [comment]
Hey @SumiBisht, could you mention steps for us to reproduce this?
A [codesandbox](https://csb.app) would work for this purpose

**Reply by SumiBisht** [reply]
This issue is coming up when I go to a certain page in my project, i can't share the sandbox i don't have rights to do that @boidushya

**Reply by SumiBisht** [reply]
<img width="833" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/40060905/971b7db0-8ab6-4e8c-b5a2-62c11eb4d60e">

**Reply by xzilja** [reply]
This happens when you double initialise `@walletconnect/modal` package. Can either happen on it's own or if you are creating something like ethereum-provider multiple times. Make sure you only create instances for each one time per your apps life cycle. To be more precise, error is due to tying to create `<wcm-button />` web component that was already created before.
